backend/followups.js

```diff
@@ -7,7 +7,9 @@ const app = express()
 const router = new express.Router()
 
 router.param("followupId", followups.followup)
+// TODO next line is to be removed once tokens are used globally
 router.get("/:followupId", followups.resultRedirect)
+router.get("/:followupId/:token", followups.resultRedirect)
```

Comment:
Je propose de supprimer l'id qui me parait inutile et en même temps de donner un nom plus explicite. La route ne me paraissait pas intuitive.
```suggestion
router.get("/followups/simulation/:token", followups.resultRedirect)
```